custom_components/evse_load_balancer/chargers/amina_charger.py

```diff
@@ -97,19 +91,74 @@ async def set_phase_mode(
-        single_phase = "disable"
-        if mode == PhaseMode.SINGLE:
-            single_phase = "enable"
-
+        single_phase = "enable" if mode == PhaseMode.SINGLE else "disable"
         await self._async_mqtt_publish(
             topic=self._topic_set, payload={AminaPropertyMap.SinglePhase: single_phase}
         )
 
     async def set_current_limit(self, limit: dict[Phase, int]) -> None:
-        """Set the charger limit."""
-        current_value = min(*limit.values(), AMINA_HW_MAX_CURRENT)
+        """
+        Set the charger limit and manage ON/OFF around the 6A hardware clamp.
+
+        For synced chargers like Amina, use the minimum across phases
+        (the charger applies the same limit to all phases). Tests and
+        behavior expect the smallest per-phase requested value to be used.
+        """
+        requested_current = min(limit.values()) if limit else 0
+
+        # Pause path: explicitly turn the charger OFF when we request below HW min
+        if requested_current < AMINA_HW_MIN_CURRENT:
+            self._LOGGER.debug(
+                "AminaCharger: requested %sA < %sA, sending OFF + charge_limit=%s",
+                requested_current,
+                AMINA_HW_MIN_CURRENT,
+                AMINA_HW_MIN_CURRENT,
+            )
+            await self._async_mqtt_publish(
+                topic=self._topic_set,
+                payload={
+                    AminaPropertyMap.State: "OFF",
+                    AminaPropertyMap.ChargeLimit: AMINA_HW_MIN_CURRENT,
+                },
+            )
+            return
+
+        # Enable path: request >= AMINA_HW_MIN_CURRENT
+        current_value = min(requested_current, AMINA_HW_MAX_CURRENT)
+        car_conn = self.car_connected()
+        ev_status = self._state_cache.get(AminaPropertyMap.EvStatus)
+
+        self._LOGGER.debug(
+            "AminaCharger: car_connected=%s ev_status=%s requested=%s",
+            car_conn,
+            ev_status,
+            requested_current,
+        )
+
+        if not car_conn:
+            # Car not connected — just publish limit
+            self._LOGGER.debug(
+                "AminaCharger: car not connected; publishing ChargeLimit only "
+                "(requested=%s)",
+                requested_current,
+            )
+            await self._async_mqtt_publish(
+                topic=self._topic_set,
+                payload={AminaPropertyMap.ChargeLimit: current_value},
+            )
+            return
+
+        # Car connected — send ON + ChargeLimit
+        self._LOGGER.debug(
+            "AminaCharger: publishing ON and ChargeLimit=%s",
+            current_value,
+        )
         await self._async_mqtt_publish(
-            topic=self._topic_set, payload={AminaPropertyMap.ChargeLimit: current_value}
+            topic=self._topic_set,
+            payload={
+                AminaPropertyMap.State: "ON",
+                AminaPropertyMap.ChargeLimit: current_value,
+            },
```

Comment:
What happens if we always send a single message with `ON` and the `ChargeLimit`? In theory the code paths in `set_current_limit` are only reached when the coordinator is actually "monitoring loads" and thus a car is connected. So reasoning about this I feel like we should be able to remove this conditio and just sed the "ON" and "Limit" value(?)